frontend/src/components/special/IdContainer.tsx

```diff
@@ -24,9 +24,11 @@ function IdContainer(props: IdProps) {
       {
         id.split('').map((c, index) => {
           if (id.length - 1 === index) {
-            return <NumberContainer style={{ margin: 0 }}>{c}</NumberContainer>;
+            // eslint-disable-next-line react/no-array-index-key
+            return <NumberContainer key={index} style={{ margin: 0 }}>{c}</NumberContainer>;
           }
-          return <NumberContainer>{c}</NumberContainer>;
+          // eslint-disable-next-line react/no-array-index-key
+          return <NumberContainer key={index}>{c}</NumberContainer>;
```

Comment:
I think you added the key to remove the warning, but you left the disable warning comment here - did you mean to remove that?

Reply:
The key removed the errors in the console, but ESLint still gives a warning for using the index as the key (it doesn't want you to since if the items in a list reorder, then using indices can be bad for performance). However I couldn't find any other option (the character `c` can be duplicate) and also this is splitting up a static string so it will never change order, so I figured it'd be ok - but lmk your thoughts

Reply:
Ok gotcha - yeah this is totally fine by me, thanks for the explanation 👍